<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="btnCalculatePageFaults">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>130</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Calculate Page Faults</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinNumPages">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>49</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinNumFrames">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>50</y>
      <width>49</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtReferenceString">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>331</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumPages">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Pages: </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNumFrames">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>141</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Number of Frames:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblRefString">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>121</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Reference String: </string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenerateReferenceString">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>90</y>
      <width>171</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Generate Ref. String</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmboAlgorithm">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>86</width>
      <height>25</height>
     </rect>
    </property>
    <property name="currentText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblAlgorithm">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>101</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Algorithm: </string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
